pkgs/applications/emulators/fceux/default.nix

```
@@ -12,13 +12,13 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "fceux";
-  version = "2.6.4";
+  version = "v2.6.6";
```

```suggestion
  version = "2.6.6";
```

Version should start with a digit